### ISSUE TITLE
Package dependency error

### ISSUE THREAD

**Issue by kirill-dev-pro** [MainRo/deepspeech-server#30]
With python 3.7 im installing `deepspeech-server` with pip
```
pip install deepspeech-server
```
while instalation it shows this errors
```
Successfully built deepspeech-server
ERROR: cyclotron 1.0.0 has requirement rx>=3.0, but you'll have rx 1.6.1 which is incompatible.
ERROR: cyclotron-std 1.0.0 has requirement rx>=3.0, but you'll have rx 1.6.1 which is incompatible.
Installing collected packages: rx, cyclotron, cyclotron-std, deepspeech-server
Successfully installed cyclotron-1.0.0 cyclotron-std-1.0.0 deepspeech-server-1.1.0 rx-1.6.1
```
after instalation if I run `deepspeech-server` im getting 
```
Traceback (most recent call last):
  File "/home/parallels/Work/deepspeech-server/deepspeech_venv/bin/deepspeech-server", line 3, in <module>
    from deepspeech_server.server import main;
  File "/home/parallels/Work/deepspeech-server/deepspeech_venv/lib/python3.7/site-packages/deepspeech_server/server.py", line 7, in <module>
    from cyclotron.router import make_error_router
  File "/home/parallels/Work/deepspeech-server/deepspeech_venv/lib/python3.7/site-packages/cyclotron/router.py", line 2, in <module>
    import rx.operators as ops
ModuleNotFoundError: No module named 'rx.operators'
```

**Comment by MainRo** [comment]
yes, this is already tracked in #23

**Comment by MainRo** [comment]
Fixed in v2.0.0
